avm/res/web/site/slot/main.bicep

```
@@ -264,6 +264,7 @@ module slot_appsettings 'config--appsettings/main.bicep' = if (!empty(appSetting
     storageAccountUseIdentityAuthentication: storageAccountUseIdentityAuthentication
     appInsightResourceId: appInsightResourceId
     appSettingsKeyValuePairs: appSettingsKeyValuePairs
+    currentAppSettings: !empty(app.id) ? list('${app.id}/config/appsettings', '2023-12-01').properties : {}
```

```suggestion
    currentAppSettings: !empty(slot.id) ? list('${slot.id}/config/appsettings', '2023-12-01').properties : {}
```

Small fix, otherwise the slot retains the settings of the main app.

---

@krbar Thanks also for pointing me to this issue to fix!

---

Ah good catch. @krbar . Thanks @peterbud for fix